pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -47,28 +46,28 @@ let
     assert forceWayland -> (browser ? gtk3); # Can only use the wayland backend if gtk3 is being used
 
     let
-      enableAdobeFlash = cfg.enableAdobeFlash or false;
       ffmpegSupport = browser.ffmpegSupport or false;
       gssSupport = browser.gssSupport or false;
       alsaSupport = browser.alsaSupport or false;
       pipewireSupport = browser.pipewireSupport or false;
 
+      # FIXME: This should probably be an assertion now?
       plugins =
         let
           removed = lib.filter (a: builtins.hasAttr a cfg) [
-            "enableVLC"
+            "enableAdobeFlash"
+            "enableAdobeReader"
+            "enableBluejeans"
             "enableDjvu"
+            "enableFriBIDPlugin"
+            "enableGoogleTalkPlugin"
             "enableMPlayer"
-            "jre"
+            "enableVLC"
             "icedtea"
-            "enableGoogleTalkPlugin"
-            "enableFriBIDPlugin"
-            "enableBluejeans"
-            "enableAdobeReader"
+            "jre"
```

Comment:
I sorted this list when I added `enableAdobeFlash`, which is why this chunk of the diff is weird.

Reply:
Do any of these still work? Those are all NPAPI plugins, right? Those haven't been working [for a while](https://support.mozilla.org/en-US/kb/npapi-plugins).

Not saying this should be part of this PR, just wondering if we should drop all of this other stuff like `enableGoogleTalk`, `enableAdobeReader`, etc. as well and if you'd be interested in doing this as a follow-up PR.

Reply:
Oops, nevermind… I didn't read enough context sorry for the noise.